_search-plugins/searching-data/filtering-search-results.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: Filter results
+parent: Searching data
+nav_order: 21
+redirect_from:
+  - /opensearch/search/filter/
+---
+# Filtering search results in OpenSearch
+
+Filtering the search results in OpenSearch allows you to refine the results returned. Specifying criteria to filter the indexes/documents by, allows you to refine the returned results. You can filter the results based on ranges, conditions or specific terms to refine the returned results. This is helpful when you have large datasets and allows you to interpret and understand larger datasets more easily.
+
+Filtering allows you to:
+- Improve search accuracy by reducing unneeded information on a case by case basis to allow accurate interpretation of results. 
+- Enhance performance by reducing the amount of data necessary to process and return when querying. Improving query performance. 
+- Categorised search by allowing data to be returned in categories which allows you to explore the data in a structured manner.
+
+## Filtering with OpenSearch Dashboards
+
+To begin filtering in OpenSearch Dashboards, follow these steps:
+
+1. Navigate to the OpenSearch Dashboards UI.
+2. Click on `Discover` in the sidebar.
+3. Choose `opensearch_dashboards_sample_data_flights` from the index pattern selector.
```

Comment:
This should be more generic and also point them to the page on how to create an index patter. A user might not have installed the flights sample dataset. By default they dont have any sample datasets installed and even if they do manually add them, they will not add all.

Reply:
Se additions at line 16